### PR TITLE
Add ability to force builds outside the aquifer project root

### PR DESCRIPTION
**Pull request by ccjjmartin**
## Purpose:
- Add ability to force builds outside the aquifer project root
## Notes:
- In certain situations, it is desirable to build outside the project root.  The `del` node module allows this but we need a way for aquifer to pass the `--force` option to the module.
## Reproduce:
- `aquifer create test`
- `cd test`
- Edit your aquifer.json file to make the build directory `../build`
- `aquifer build`
## Error:
- Drush make works fine but "overwriting" symlinks doesn't work if the build directory is outside the project root.  I believe this is new with the upgrade of the dependency "fs-extra": "^0.30.0", from "fs-extra": "^0.18.0".  It used to work just fine.

```
Overwriting: /Users/developer/repos/build/.htaccess
[Error: Cannot delete files/folders outside the current working directory. Can be overriden with the `force` option.]
```
- Most people recognize the force command as something potentially dangerous which is why I chose force.

### CODE REVIEW

**Comment by JeffTomlinson** [comment]
@ccjjmartin Can you include steps to test?

**Comment by ccjjmartin** [comment]
@JeffTomlinson: Steps added to the PR notes

**Comment by ccjjmartin** [comment]
I also chose force because that is what the error message says to do but aquifer doesn't have a force option ... yet.

**Comment by patrickocoffeyo** [comment]
The "consequences" of building outside of the root directory aren't dangerous in any way imho, I think it'd be fine if we just always allowed del to delete build files that are outside of the root.

**Comment by ccjjmartin** [comment]
@patrickocoffeyo Want me to redo this PR with "force" set as the default all the time?

**Comment by ccjjmartin** [comment]
@patrickocoffeyo I have updated this PR to set "force" as the default so builds can be done outside of the project root without the force flag.

File: lib/build.api.js [hunk 1]
```diff
@@ -192,7 +192,8 @@ class Build {
             switch (link.conflict) {
               case 'overwrite':
                 this.aquifer.console.log('Destination exists. Conflict set to overwrite. \nOverwriting: ' + link.dest, 'status');
-                del.sync(link.dest);
+                options.force = true;
```
Comment (JeffTomlinson) [inline lib/build.api.js]: @ccjjmartin This looks like it is altering the options that are passed into the command and therefore may contain properties unrelated to `del.sync()`. Should we define a new object here instead to pass to `del.sync()`?

Comment (ccjjmartin) [inline lib/build.api.js]: Great catch.  At some point, I had tried to include this in aquifer.json file but that is no longer necessary.  I marked this as needs work and will update soon.

**Comment by ccjjmartin** [comment]
Removed potential conflict with options object.  Great catch.  This should be ready for another round of reviews.

**Comment by patrickocoffeyo** [comment]
@ccjjmartin :shipit:!

**Pull request merged**